### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by richardcase**
Added multiple templates for the issues:
- Bug
- Feature Request
- Help

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+**What happened?**
+A description of actual behvaior (with error messages).
+
+**What you expected to happen?**
+A clear and concise description of what the bug is.
+
+**How to repoduce it?**
+Steps to reproduce the behavior:
+1. Go to '...'
```
Comment (errordeveloper) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe we could say something more generic here? Maybe just leave each of the steps blank?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Anything else we need to know?**
+Hardware, OS and screenshots (if applicable)
```
Comment (errordeveloper) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I would take out 'Hardware' and 'screenshots'. Perhaps we could ask about the OS, whether they installed binary or compiled it, and how are their AWS credentials configured (namely MFA and stuff). Wdyt?

Comment (richardcase) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah i think thats a good idea, i have changed.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (richardcase) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I've changed this.

**Comment by richardcase** [comment]
@errordeveloper - it is for the latest GitHub thing to prompt for the issue type. I know there is automation with projects but i think that is to just assign it to a column. We could add a bot that assigns the label?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a bug report to help us improve eksctl
+
+---
+
+**What happened?**
+A description of actual behvaior (with error messages).
```
Comment (dholbach) [inline .github/ISSUE_TEMPLATE/bug_report.md]: *behavior*

**Comment by richardcase** [comment]
@dholbach  - thanks for spotting the typo. I have fixed that and i've now installed a spell checker into vscode to help with my spelling! So there should be less going forward.

**Pull request merged**